ep0099/ep0099.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package ep0099
+
+import (
+	"errors"
+
+	"periph.io/x/conn/v3/i2c"
+)
+
+var InvalidAddressError = errors.New("Invalid EP-0099 address")
```

Comment:
invalid (lower case)

Reply:
after analyzing github actions build go-lint output, it suggests that this is renamed to "ErrInvalidAddress".
I will change it to that and make it lower case so it's internal to the package.